iso-19115-19119/srv-linkage.md

```diff
@@ -43,6 +43,8 @@ Since WCS and SOS are now also supported by technical guidance documents, the ET
 
 The namespace prefixes used as described in [README.md](http://inspire.ec.europa.eu/id/ats/metadata/1.3/iso-19115-19119/README#namespaces).
 
+The metadata validator should accept 'text/xml', 'application/xml', 'application/vnd.ogc.*' and 'application/atom' as a media type for distribution.
```

Comment:
The media type check should be included in the test section, after "If one or more are provided: For each linkage the test checks, if the linkage element contains an element of type gmd:URL. Retrieve the resource at the URL using HTTP GET." At this point, a check for the media type should be added.